projects/plugins/jetpack/extensions/blocks/ai-image/index.js

```diff
@@ -19,21 +19,32 @@ export const settings = {
 		<Fragment>
 			<p>
 				{ __(
-					'Automatically generate an illustration for your post, powered by AI magic. We are experimenting with this feature and can tweak or remove it at any point.',
+					'Automatically generate an illustration for your post, powered by AI magic.',
 					'jetpack'
 				) }
 			</p>
+			<p>
+				{ __(
+					'We are experimenting with this feature and can tweak or remove it at any point.',
+					'jetpack'
+				) }
+			</p>
+			<ExternalLink href="https://en.forums.wordpress.com/?post_type=topic&p=3907816">
+				{ __( 'Share your feedback.', 'jetpack' ) }
+			</ExternalLink>
 		</Fragment>
 	),
 	icon: {
-		src: 'welcome-write-blog',
+		src: 'superhero',
 		foreground: getIconColor(),
 	},
 	category: 'media',
 	keywords: [
 		_x( 'AI', 'block search term', 'jetpack' ),
+		_x( 'AL', 'block search term', 'jetpack' ),
```

Comment:
This may be a stupid question, but does "AL" stand for? Should we add more info in the context? I assume I won't be the only one who doesn't know :)

Reply:
I will post @jobthomas feedback that led to this:

> This is pretty awesome. As the tiniest of comments, and not sure how to fix it. Given the font of the block name, I think several of our users may read this as “AL IMAGE” rather than “AI IMAGE” (using all caps to avoid confusion). I’m not sure what would be a good way to fix that though.


If you think this is harmful, I can remove this change :)